api/job.pb.go

```diff
@@ -5,7 +5,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.33.0
-// 	protoc        v4.24.4
+// 	protoc        v4.25.4
```

Comment:
can this diff be removed? [We arent using this version in master currently](https://github.com/diagridio/go-etcd-cron/blob/master/Makefile#L39)
```suggestion
// 	protoc        v4.24.4
```

Reply:
we are updating it in the other PR in this repo and in dapr so Im fine with this I suppose